### PR TITLE
Added fix for query string check

### PR DESCRIPTION
**Pull request by TWallace**
Created this PR on behalf of Muthu, since he doesn't have rights to create the PR himself.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ function prepareBody(opts) {
 function prepareUriObj(opts) {
     // if uri is a regexp, this will not support qs params since ? marks an optional character in regexp
     let queryIndex = opts.uri instanceof RegExp ? -1 : opts.uri.indexOf('?');
-    if (queryIndex !== -1) {
+    if (queryIndex !== -1 && !opts.qsString) {
```
Comment (andyday) [inline index.js]: what's the use case for this?

Comment (Smuthuraman) [reply index.js]: We have a offer skus endpoint which parses the querystring as a string and rejects the parsed querstring from vhttp module. So adding this condition helps to send the querystring as its passed to vhttp module.

**Comment by andyday** [comment]
@tmwllc do you have access to publish this to npm?

**Comment by tmwllc** [comment]
Yes I do, I'll publish.

**Pull request merged**